### ISSUE TITLE
Notice: GitHub Actions Replacement

### ISSUE THREAD

**Issue by banesullivan** [pyvista/gl-ci-helpers#8]
For a cross-platform compatible GitHub Action, see https://github.com/marketplace/actions/setup-headless-display-action

Source: https://github.com/pyvista/setup-headless-display-action

**Comment by larsoner** [comment]
Nice! Still need this repo for Azure and (more rarely perhaps) AppVeyor, though, right?

**Comment by banesullivan** [comment]
That's right! We should keep this here incase downstream folks are still using Azure, etc, but I've moved PyVista's CI entirely to GitHub Actions... still need to migrate PyVistaQt and other packages though
